build/webpack.config.js

```diff
@@ -24,90 +23,74 @@ if (fs.existsSync('build/rsync.json')) {
 const lib = path.resolve('src/lib');
 const thirdParty = path.resolve('src/third-party');
 const staticFolder = path.resolve('dist');
-
-const languages = isRelease
-    ? locales
-    : ['en-US']; // Only 1 language needed for dev
+const languages = isRelease ? locales : ['en-US']; // Only 1 language needed for dev
 
 /* eslint-disable key-spacing, require-jsdoc */
 function updateConfig(conf, language, index) {
     const config = Object.assign(conf, {
         entry: {
             annotations: ['box-annotations'],
             preview: [`${lib}/Preview.js`],
-            csv: [`${lib}/viewers/text/BoxCSV.js`]
+            csv: [`${lib}/viewers/text/BoxCSV.js`],
+        },
+        mode: isRelease ? 'production' : 'development',
+        optimization: {
+            minimizer: [
+                new UglifyJsPlugin({
```

Comment:
I had read that TerserPlugin (which I think comes by default) is preferred. Should we switch to that instead?

Reply:
I've seen that as well, but it seemed beyond the scope of what I was trying to achieve. It seems to work fine and other repos all still use UglifyJsPlugin, so I'm going to leave it as-is for now.

Reply:
👍